### PR TITLE
update release notes for 0.7.2

### PR DESCRIPTION
**Pull request by cyrush**
@xjrc  can you review my entry to the release notes about the change to `blueprint::verify`

### CODE REVIEW

**Comment by xjrc** [comment]
@cyrush: Will try to do this in a bit.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ and this project aspires to adhere to [Semantic Versioning](https://semver.org/s
 - Updated to newer version of BLT to leverage CMake's FindMPI defined targets when using CMake 3.15 or newer.
 - Changed `rapidjson` namespace to `conduit_rapidjson` to avoid symbol collisions with other libraries using RapidJSON.
 
+#### Blueprint
+- The semantics of `conduit::blueprint::mesh::verify` changed. An empty conduit Node is now considered a valid multi-domain mesh with zero domains. If you always expect mesh data, you can add an additional check for empty and to craft code that works for both the old and new verify semantics.
```
Comment (xjrc) [inline CHANGELOG.md]: ...you can add an additional check for empty ~and~ to craft code...

**Review by xjrc — APPROVED** [review]
Looks good; just one small typo to fix.

File: CHANGELOG.md (hunk 1 above)
Comment (cyrush) [reply CHANGELOG.md]: thanks!

**Pull request merged**